src/components/pages/dashboard/panels/telemetry/telemetryPanel.js

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+import { Observable } from 'rxjs';
+// import update from 'immutability-helper';
+import 'tsiclient';
+
+import { TelemetryService } from 'services';
+import { Indicator } from 'components/shared';
+import {
+  Panel,
+  PanelHeader,
+  PanelContent,
+  PanelOverlay
+} from 'components/pages/dashboard/panel';
+import { joinClasses } from 'utilities';
+
+import './telemetryPanel.css';
+// TODO: find a way to import without the relative path
+import '../../../../../../node_modules/tsiclient/tsiclient.css';
+
+const telemetryChartId = 'device-telemetry-chart';
+
+const chartColors = [
```

Comment:
May be add actual color names in comments for clarity

Reply:
I don't know how helpful that will actually be. We don't have names from design and even then it will be stuff like 'stone' or 'ash'. These colors will be moved to a config later on so the redux store can switch them out when the theme changes.